vigiliae-christianae.csl

```diff
@@ -0,0 +1,790 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="chicago" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Vigiliae Christianae</title>
+    <title-short>VC</title-short>
+    <id>http://www.zotero.org/styles/vigiliae-christianae</id>
+    <link href="http://www.zotero.org/styles/vigiliae-christianae" rel="self"/>
+    <link href="http://www.zotero.org/styles/society-of-biblical-literature-fullnote-bibliography" rel="template"/>
+    <link href="http://www.brill.com/sites/default/files/ftp/authors_instructions/VC.pdf" rel="documentation"/>
+    <author>
+      <name>Carl Johan Berglund</name>
+      <email>cajo@cajo.nu</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="theology"/>
+    <issn>0042-6032</issn>
+    <eissn>1570-0720</eissn>
+    <summary>Vigiliae Christianae</summary>
+    <updated>2016-10-06T09:13:00+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="editor" form="verb-short">ed.</term>
+      <term name="translator" form="verb-short">trans.</term>
+      <term name="at">here</term>
+    </terms>
+  </locale>
+    <locale xml:lang="de">
+    <terms>
+      <term name="editor" form="verb-short">Hrsg.</term>
+      <term name="translator" form="verb-short">Übers.</term>
+      <term name="at">hier</term>
+    </terms>
+  </locale>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="editor" form="verb-short">éd.</term>
+      <term name="translator" form="verb-short">trad.</term>
+      <term name="at">ici</term>
+    </terms>
+  </locale>
+<locale>
+	  <terms>
+  		<term name="at">hic</term>
+  	</terms>
+  </locale>
```

Comment:
@carljohanberglund, did you leave in lines 43-47 intentionally? (that define `<term name="at">hic</term>`)